google-cloud-storage/lib/google/cloud/storage/file.rb

```diff
@@ -986,6 +993,11 @@ def download path = nil, verify: :md5, encryption_key: nil, range: nil,
         # @param [String] encryption_key Optional. The customer-supplied,
         #   AES-256 encryption key used to encrypt the file, if one was provided
         #   to {Bucket#create_file}.
+        # @param [Boolean] force_copy_metadata Optional. If `true` and if updates
+        #   are made in a block, the metadata for the source object will be copied
+        #   in its entirety to the destination object. Otherwise, only the updates
+        #   made in the yielded block will be applied to the destination object,
+        #   and its other metadata fields will be empty. The default is `nil`.
```

Comment:
I want to make sure, ACLs continue not to be copied.

Reply:
I added an assertion to the acceptance test ensuring that the ACL is not copied, and it currently fails. It seems to me that the safest solution is probably a whitelist of fields that should be copied, rather than excluding fields such as `acl`. Here is my proposed whitelist:

* cache_control
* content_disposition
* content_encoding
* content_language
* content_type
* metadata

@frankyn @renchap What do you think?

Reply:
Clarifying, metadata is custom metadata. 
This is a good idea, thanks @quartzmo. We want to be careful not to copy all metadata especially ACL.

Reply:
OK, thanks @frankyn, I'll go ahead with this whitelist.

Reply:
Updated, PTAL.